### PR TITLE
fix: missing annotations on EdxUserSchoolRole

### PR DESCRIPTION
**Pull request by trev-dev**

### CODE REVIEW

File: api/src/main/java/ca/bc/gov/educ/api/edx/struct/v1/EdxUserSchoolRole.java [hunk 1]
```diff
@@ -13,8 +16,14 @@
 public class EdxUserSchoolRole extends BaseRequest implements Serializable {
   private static final long serialVersionUID = 583620260139143932L;
 
+  @NotNull(message = "edxSchoolRoleID cannot be null.")
```
Comment (SoLetsDev) [inline api/src/main/java/ca/bc/gov/educ/api/edx/struct/v1/EdxUserSchoolRole.java:19]: should be edxUserSchoolRoleID?

File: api/src/main/java/ca/bc/gov/educ/api/edx/struct/v1/EdxUserSchoolRole.java [hunk 2]
```diff
@@ -13,8 +16,14 @@
...
   String edxUserSchoolRoleID;
+
+  @NotNull(message = "edxRoleCode cannot be null.")
+  @Size(max = 32, message = "edxRoleCode should be no longer than 32 characters.")
   String edxRoleCode;
+
+  @NotNull(message = "edxRoleCode cannot be null.")
```
Comment (SoLetsDev) [inline api/src/main/java/ca/bc/gov/educ/api/edx/struct/v1/EdxUserSchoolRole.java:26]: I think you meant edxUserSchoolID here

File: api/src/main/java/ca/bc/gov/educ/api/edx/struct/v1/EdxUserSchoolRole.java (hunk 1 above)
Comment (trev-dev) [reply api/src/main/java/ca/bc/gov/educ/api/edx/struct/v1/EdxUserSchoolRole.java]: Got it

File: api/src/main/java/ca/bc/gov/educ/api/edx/struct/v1/EdxUserSchoolRole.java (hunk 2 above)
Comment (trev-dev) [reply api/src/main/java/ca/bc/gov/educ/api/edx/struct/v1/EdxUserSchoolRole.java]: Yep

**Review by SoLetsDev — APPROVED** [review]

**Pull request merged**